src/main.js

```diff
@@ -5,9 +5,11 @@ import Vue from 'vue'
 import App from './App'
 import router from './router'
 import store from './store'
-import jsPDF from 'jspdf' 
-import html2canvas from "html2canvas"
-
+// import jsPDF from 'jspdf' 
```

Comment:
Maybe needs to be put back in.
```suggestion
// import jsPDF from 'jspdf' 
// import html2canvas from "html2canvas"
```